docs/tutorials/further_topics/fingerprinting.rst

```diff
@@ -77,5 +80,49 @@ This allows for the fingerprinting of parametrized strategies::
     >>> data[(0, 0)]
     4.4...
 
-Ashlock's fingerprint is currently the only fingerprint implemented in the
-library.
+Transitive Fingerprints
+-----------------------
+
+Another implemented fingerprint is the transitive fingerprints. The
```

Comment:
typo: 'is the transitive fingerprints' should be 'is the transitive fingerprint'